src/calculateRentalCost.js

```diff
@@ -3,8 +3,23 @@
  *
  * @return {number}
  */
+
 function calculateRentalCost(days) {
-  // write code here
+  const longTerm = 7;
+  const longTermDiscount = 50;
+  const smallTerm = 3;
+  const smallTermDiscount = 20;
+  const valueDay = 40;
+
+  if (days >= longTerm) {
+    return days * valueDay - longTermDiscount;
+  }
+
+  if (days >= smallTerm) {
+    return days * valueDay - smallTermDiscount;
```

Comment:
Similarly, the discount logic for rentals of 3 days or more applies a fixed discount of 20. Verify if this is the intended behavior according to the task requirements. If the discount should be calculated differently, consider revising this logic.